### PR TITLE
Chore: Default font-weight

### PR DESCRIPTION
**Pull request by WissenIstNacht**
This PR changes the default fontweight of the ui-components and the app from 400 to 500.

This is done because the default font-weights of the design is 500. HTML on the other hand as a default of 400. This means that every instance of non-bold text must be modified with the `font-medium` TW class. This is cumbersome and easy to forget.

I have not found a way to do this in TW, unfortunately.

Also, I have not refactored the web-app, since I didn't want to cause any conflicts.

### CODE REVIEW

File: packages/web-app/index.html [hunk 1]
```diff
@@ -1,61 +1,66 @@
...
-          rudderAnalytics.page();
+  <!-- TODO: remove once fonts have been downloaded and set in stone -->
+  <link
+    href="'https://fonts.googleapis.com/css2?family=Manrope:wght@500;700&display=swap'"
+    rel="stylesheet" />
+  <style>
+    p {
+      font-weight: 500;
+    }
+  </style>
```
Comment (WissenIstNacht) [inline packages/web-app/index.html]: These are the only actual changes in this file. The rest is just formatting.

**Review by sepehr2github — APPROVED** [review]
Thank you, Vali. Looks good to me. the only thing we should care about is to remember adding the API key later.

File: packages/web-app/index.html (hunk 1 above)
Comment (sepehr2github) [reply packages/web-app/index.html]: btw Vali, Don't you think we should add this font-weight for the 'span' element?

Comment (WissenIstNacht) [reply packages/web-app/index.html]: Eeeeeeexcellent point good point, thanks! 👍🏼

**Review by RakeshUP — APPROVED** [review]

**Comment by WissenIstNacht** [comment]
@sepehr2github I just checked regarding the key. Apparently, this was done moths ago 😅

I'll delete the TODO and merge the PR :)

**Pull request merged**